modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -71,6 +71,14 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following bug fixes:
 
+=== On iOS Safari, Korean characters will no longer merge onto the previous line after inserting a newline by pressing Enter.
```

Comment:
If we're trying to match the title exactly to the changelog entry, this is the latest version.
```suggestion
=== On iOS Safari, Korean characters were merging onto the previous line upon typing after inserting a newline by pressing Enter.
```